docs/upgrade-notes/update-cumulus_id-type-indexes-CUMULUS-3449.md

```diff
@@ -0,0 +1,176 @@
+---
+id: update-cumulus_id-type-indexes-CUMULUS-3449
+title: Update Cumulus_id Type and Indexes
+hide_title: false
+---
+
+## Background
+
+As part of the work for [CUMULUS-3449](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3449), LPDAAC have identified some Cumulus Core changes in
+regards to the database in order to improve query performance. For other recommendations and considerations, see LPDAAC wiki
+[Cumulus RDS Index testing](https://wiki.earthdata.nasa.gov/pages/viewpage.action?spaceKey=LPCUMULUS&title=Cumulus+RDS+Index+testing).
+
+The following updates are included:
+
+- Update some cumulus_id columns to BIGINT to allow future data grows and fix data type mismatch between primary key and foreign key. The columns are:
+  - executions.cumulus_id
+  - executions.parent_cumulus_id
+  - files.granule_cumulus_id
+  - granules_executions.granule_cumulus_id
+  - granules_executions.execution_cumulus_id
+  - pdrs.execution_cumulus_id
+
+- Update indexes to provide the best chances of joins between tables
+  - Change granules table unique constraint from
+    `granules_granule_id_collection_cumulus_id_unique UNIQUE (granule_id, collection_cumulus_id)`
+    to
+    `granules_collection_cumulus_id_granule_id_unique UNIQUE (collection_cumulus_id, granule_id)`
+  - Add indexes `granules_granule_id_index` and `granules_provider_collection_cumulus_id_granule_id_index`
+    to `granules` table
+
+The updates will be automatically created as part of the bootstrap lambda function on deployment of the data-persistence module.
+
+*In cases where the type update and indexes are already applied, the updates will have no effect.*
+
+## Apply the Changes in Production Environment
+
+In production and some UAT environments, the type updates and indexes must be applied manually since the commands can take significant
```

Comment:
Suggest we make the distinction between 'large' and 'production'.    Users are going to want to know what row count (approximately) given DB size they can run the bootstrap versus this sort of apply, we should make it very clear what the difference is, or suggest any non-new deployment should use this procedure instead.